src/odemis/driver/technolution.py

```diff
@@ -1548,16 +1554,20 @@ def __init__(self, url, response, expected_status):
             self._errorMessageResponse(url, status_code, reason, expected_status,
                                        content_translated['status_code'],
                                        content_translated['message'])
+
+            # Also log the error so it is easier to find it back when the error was received in the log
+            logging.error(self._error)
```

Comment:
No, please move it to last line of the function (so that it logs all the errors). Just not inside a `finally` block.